app/signoz-expert-contributors-program/page.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import ReactMarkdown from 'react-markdown'
+
+import { Metadata } from 'next'
+
+export const metadata: Metadata = {
+  title: 'SigNoz Program for Documentation',
+}
+
+const markdownContent = `
+
+# SigNoz Expert Contributors Program
+
+The SigNoz Expert Contributors Program is an exclusive initiative for seasoned professionals in the observability domain. 
+If you're an expert eager to showcase your skills, contribute to a leading open-source project, and get compensated for your expertise, this program is for you.
+
+![Create dashboards and documentation for SigNoz with its Expert Contributors program, build your digital presence and get paid to do it](/img/ecp-cover.webp)
+
+&nbsp;
+
+## Why join the program ?
+
+**Monetary Compensation:** Receive competitive payments for your high-quality contributions to SigNoz.
+
+**Exclusive Badges:** Earn SigNoz Expert Badges as proof of your work and expertise. Display them on your professional profiles, resumes, or portfolios.
```

Comment:
```suggestion
**Exclusive Badges:** Earn SigNoz Expert Badges as proof of your work and expertise. Display them on your professional profiles, resumes, or portfolios.
```

How are we planning to do this? How will someone display the badge on sth like linkedin? Let's not mention this if there is no clear plan.

Reply:
> How are we planning to do this? How will someone display the badge on sth like linkedin? Let's not mention this if there is no clear plan.

Was planning to do this in a similar way that PostHog does. This will require us to create a page for it and have links for individual contributors which they can use on LinkedIn or anywhere else they want.

Removing this for now. Will pick it up in the second interaction.